src/cairoUtilFuncGen/index.ts

```diff
@@ -248,6 +248,11 @@ export class CairoUtilFuncGen {
       })
       .join('\n\n');
   }
+  removeGeneratedFunction(name: string) {
+    this.getAllChildren()
+      .filter((c) => c instanceof StringIndexedFuncGen)
+      .forEach((c) => (c as StringIndexedFuncGen).removeGeneratedFunction(name));
```

Comment:
Wouldn't `(c: StringIndexedFuncGen)` work without an explicit cast? :thinking:

Reply:
`(c: StringIndexedFuncGen)` throws invalid cast error. It seems I cannot use `CairoUtilFuncGenBase` object as a `StringIndexedFuncGen` without the explicit cast.